examples/environment_config/real_example.config.json

```diff
@@ -1,107 +1,104 @@
 {
-    "Mesh_info": {
-        "Region": {
-            "latMin": -80,
-            "latMax": -40,
-            "longMin": -130,
-            "longMax": 0,
-            "startTime": "TODAY - 4",
-            "endTime": "TODAY - 1",
-            "cellWidth": 5,
-            "cellHeight": 2.5
+    "Region": {
+        "latMin": -80,
+        "latMax": -40,
+        "longMin": -130,
+        "longMax": 0,
+        "startTime": "TODAY - 4",
+        "endTime": "TODAY - 1",
+        "cellWidth": 5,
+        "cellHeight": 2.5
+    },
+    "Data_sources": [
+        {
+            "loader": "GEBCO",
+            "params": {
+                "downsample_factors": [
+                    5,
+                    5
+                ],
+                "data_name": "elevation",
+                "file" : "datastore/bathymetry/gebco/gebco_global.nc",

```

Comment:
We are referencing local files here to show the real example. Maybe remove the real files here and have the real data examples only in the `AMOP_RoutePlanning_Examples` repo?

Reply:
I like this idea. We should include at least a README.md file with links to where to collect the data though